### PR TITLE
FOGL-6670 Addition of purge on demand to buiding pipelines section

### PR DESCRIPTION
**Pull request by MarkRiddoch**
Signed-off-by: Mark Riddoch <mark@dianomic.com>

### CODE REVIEW

File: docs/building_pipelines.rst [hunk 1]
```diff
@@ -56,10 +59,30 @@ The Fledge pipeline mechanism is designed for and lends itself to a modular deve
...
++----------------+
+| |manual_purge| |
++----------------+
+
+.. image:: images/eraser.jpg
+     :align: left
+
+These icons are resemble erasers and are located in each row of the assets and also in the top right corner next to the help icon. Clicking on the eraser icon in each of the rows will purge the data for just that asset, leaving other assets untouched. Clicking on the icon in the top left corner will purge all the assets currently in the data store.
+
+In both cases a confirmation dialogue will be displayed to ensure against accidental use. If you choose to proceed the selected data within the Fledge buffer, either all or a specific asset, will be erased. There is no way to undo this operation or to retrieve the data once it has been purged.
```
Comment (ashish-jabble) [inline docs/building_pipelines.rst:84]: dialog

File: docs/building_pipelines.rst [hunk 2]
```diff
@@ -56,10 +59,30 @@ The Fledge pipeline mechanism is designed for and lends itself to a modular deve
...
+Amongst the extra features introduced by selecting *Developer Features* will be the ability to manually purge data from the Fledge data store. This on-demand purging can be either applied to a single asset or to all assets within the data store. The manual purge operations are accessed via the *Assets & Readings* item in the Fledge menu. A number of new icons will appear when the *Developer Features* are turned on, one per asset and one that impacts all assets. 
+
++----------------+
+| |manual_purge| |
++----------------+
+
+.. image:: images/eraser.jpg
+     :align: left
+
+These icons are resemble erasers and are located in each row of the assets and also in the top right corner next to the help icon. Clicking on the eraser icon in each of the rows will purge the data for just that asset, leaving other assets untouched. Clicking on the icon in the top left corner will purge all the assets currently in the data store.
```
Comment (ashish-jabble) [inline docs/building_pipelines.rst:82]: right corner close to the help icon

Comment (MarkRiddoch) [reply docs/building_pipelines.rst:82]: Whoops, thanks

File: docs/building_pipelines.rst (hunk 1 above)
Comment (MarkRiddoch) [reply docs/building_pipelines.rst:84]: Kind of depends which side of the Atlantic you are on - but I think we Norway use the American spelling of this

**Review by ashish-jabble — APPROVED** [review]

**Pull request merged**